applications/registration-api/src/main/java/no/dcat/service/ApiCatalogHarvesterService.java

```diff
@@ -66,11 +65,15 @@ public void initQueue() {
 
         //Add them to the queue
         for (ApiCatalog catalog : allApiCatalogs) {
-            QueuedTask task = new HarvestSingleCatalogTask(catalog);
```

Comment:
Hvorfor lage en egen funksjon for bare 2 linjer ?

Reply:
Fordi jeg vil også kalle det fra post handler